app.js

```diff
@@ -13,5 +13,6 @@ app.get('/api/topics', getTopics);
 app.get('/api/articles/:article_id', getArticle);
 
 app.all('*', handleFourZeroFour);
+app.use(handlePsqlErrors);
```

Comment:
we also need to complete this error handling chain - if the error that gets passed to handlePsqlErrors doesnt get handled, and gets passed to next() - there should be another handler that handles custom errors, e.g. promise that are rejected with a 404 or 400, and then a final next() that passes any unhandled errors to a final error handler which send a 500 status code